packages/google_maps_flutter/google_maps_flutter/example/integration_test/google_maps_test.dart

```diff
@@ -1161,6 +1161,259 @@ void main() {
       expect(tileOverlayInfo1, isNull);
     },
   );
+
+  /// Account for double precision inaccuracies introduced by the platform.
+  List<WeightedLatLng> roundHeatmapData(List<WeightedLatLng> data) {
```

Comment:
See moreOrLessEquals, which handles this within the expectations themselves.

Reply:
I thought there might have been something existing to handle than. Thanks!